lib/coremget.go

```diff
@@ -19,25 +19,49 @@ import (
 // MGet allows the caller to get multiple documents based on an index, type (optional) and id (and possibly routing).
 // The response includes a docs array with all the fetched documents, each element similar in structure to a document
 // provided by the get API.
-// see http://www.elasticsearch.org/guide/reference/api/multi-get.html
+// see https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-multi-get.html
+/*
+{
+  "docs" : [
+    {
+      "_type" : "type",
+      "_id" : "1"
+    },
+    {
+      "_type" : "type",
+      "_id" : "2"
+    }
+  ]
+}
+*/
 func (c *Conn) MGet(index string, _type string, mgetRequest MGetRequestContainer, args map[string]interface{}) (MGetResponseContainer, error) {
-	var url string
+	body, err := c.DoCommand("GET", mGetUrl(index, _type), args, mgetRequest)
+	return mGetHandleResponse(body, err)
+}
+
+// MGetShort allows to get multiple documents based on an index, type (optional) and id (and possibly routing).
+// The response includes a docs array with all the fetched documents, each element similar in structure to a document
+// provided by the get API.
+// see https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-multi-get.html
+/*
+{
+	"ids": ["1", "2", "3"]
+}
+*/
+
+func (c *Conn) MGetShort(index string, _type string, mgetRequestShort MGetRequestShort, args map[string]interface{}) (MGetResponseContainer, error) {
+	body, err := c.DoCommand("GET", mGetUrl(index, _type), args, mgetRequestShort)
+	return mGetHandleResponse(body, err)
+}
+
+func mGetHandleResponse (responseBody []byte, err error) (MGetResponseContainer, error) {
```

Comment:
the mBlahBlah thing is not standard go and detracts from the readability of the code.  How about getHandleResponse?

Also, docs are helpful:

//getHandleResponse from the given response body